recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "rich" %}
-{% set version = "13.3.5" %}
+{% set version = "13.7.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 2d11b9b8dd03868f09b4fffadc84a6a8cda574e40dc90821bd845720ebb8e89c
+  sha256: 9be308cb1fe2f1f57d67ce99e95af38a1e2bc71ad9813b0e247cf7ffbcc3a432
 
 build:
   number: 1
```

```suggestion
  number: 0
```
Because it's a new version